src/app/sell/sell.class.ts

```diff
@@ -0,0 +1,43 @@
+import { iSell } from './../sell/sell.interface';
+import { iCake } from './../cake/cake.interface';
+import { dbConn } from './../app.database';
+import { SellService } from './sell.service';
+
+export class CakeSell implements iSell {
+  private sellService: SellService;
+
+  customerName: string;
+  customerPhoneNumber: string;
+  customerEmail: string;
+  totalAmount: number;
+  cake: iCake;
+
+  constructor(sell: iSell) {
```

Comment:
Here the same comment for the injectable services 😊

Reply:
Perfect! Thanks :)